### ISSUE TITLE
Screen readers cannot detect VSCode when switching to another window in VSCode

### ISSUE THREAD

**Issue by destroyersboy** [microsoft/vscode#74808]
- VSCode version: 1.34.0
- WinOs version: 1903
- DisableAllExtensions: yes

In this question, my screen reader (NVDA) doesn't react and stays on when I switch to VSCode while in another window in windows.

Although I tried to get the menus to detect VScode, this did not happen.

I'm a developer of Python and JS, I'm blind. I want to take advantage of VSCode's innovations.
Projects are now being developed in the VSCode environment. I can't adapt and feel excluded.

Another problem is that the screen readers on the code-writing screen cannot fully detect the content. The cursor is not working properly.
### Other
It can be from HTML5 aria tags. This was thought in the NVDA community we discussed. There are different problems in perception.

As a VSCode team, we cannot analyze the other problem without fixing the first issue.

What are the executives and main developers of the team think about this, what is our road map?
The problem was from the first releases. Hopefully you will find a solution in the coming months.

If there is one thing that needs to be done, you can find the support to the fullest on testing and advice.

And I'm sorry. The main language is not English.
Yours.

**Comment by isidorn** [comment]
@destroyersboy hi, thanks for using VSCode and for filling an issue.
We usually prefer having seperate vscode problems per issue created on GitHub - since it makes it easier to tackle and follow the exact problem. We also usually need reproducable steps in order to fix some issues. So let's try to distill this and figure out what is going on

1. The NVDA doesn't react when switching windows. How do you switch windows? If I just switch using alt + tab the VSCode data gets nicely read out to me
2. How did you try to detect the menus? If I press `alt` I get focused on the main menu and I can navigate left and right to get them read out
3. Cannot fully detect the content? Can you please provide more details? What content do you have and what content does the screen reader read?
4. I appologize but I did not fully understand the **Other** part. Can you please clarify what is missing on the vscode side and what you would like to see?

Thank you very much

**Comment by destroyersboy** [comment]
I'm new to github. I forgot to detail the stages.

1. I replace the windows with alt + TAB.
If I have to illustrate. I wandered through the desktop, entered a folder. Then I wanted to go back to VSCode. I'm pressing ALT + TAB.
In the incoming window I hear the name of VSCODE.
With arrows, I cannot read any content with VSCode's own shortcuts.
I'm just a screen reader who speaks the name of the window.

2. Open the menus by pressing ALT.
This sometimes works in other applications. If NVDA does not detect it. Applications designed with QT can be given as examples.
I tried it on VScode, didn't.

3. I cannot analyze the other problem.
The other problem is in the code writing section.

Simply speaking, NVDA dual-voice when writing code with the word.

Imagine that I have a simple code.

```
import os, path, randiom
```
I assume that I made a mistake in the last part of the code. When I find the wrong line, I want to find and delete the wrong part with CTRL + right arrow / CTRL + Left arrow.
NVDA's statement is as follows:
CTRL + right arrow: import, CTRL + Right arrow: import, CTRL + Right arrow: os, CTRL + Right arrow: os ...
Each word is spoken twice.
When I check the cursor, I see the cursor coming to the last letter in the second voice. "import": "t", "os" "s".
It's not the other word.
This state is not valid only in the code that is written in spacing. "Dots, commas, parentheses ..."

I'd like to mention it again. Since I cannot focus on the content of VSCode, I cannot explain this problem in detail.

Freezes during window passage, unclear.
When I switch from VSCode to another window, sometimes I don't have a problem if I left the cursor in the Write code in VSCode section. I can read the contents when I get back.
But as an example I'm navigating through the menus, I'm going to another window out of VSCode, then back to VSCode.
The first case is happening every time. I can't read inside.

Yours.

**Comment by rbenson** [comment]
@isidorn I am guessing #2 is in regards to the electron bug about menus. I couldn't find the ID number /

**Comment by isidorn** [comment]
1. Do you have an editor open in this case? Becuase if you do when I alt + tab to vscode I nicely get focuesd into the editor and I get teh content read out
2. Should be fixed since we introduced custom menus on windows. Can you reproduce @sbattern this issue under 2. that @destroyersboy is saying

**Comment by destroyersboy** [comment]
1. No. Another editor is not open.

Could it be from other dependencies of Windows or from a special setting of NVDA?
Nothing comes to my mind. If you have any other system information you want, I can tell them.
Then I'm sorry to say "no" to problem one.

2. I did not understand what you said :-).
I guess it's not about me :-).

**Comment by isidorn** [comment]
Actually duplicate of https://github.com/microsoft/vscode/issues/28316
